client-js/app/test/index.html

```diff
@@ -1,22 +0,0 @@
-<!doctype html>
```

Comment:
This came in with the polymer example project. I'm glad you've deleted it; if we ever need this it's no problem to get it back.